doc/single-nic-vm-host.md

```diff
@@ -0,0 +1,194 @@
+# How to configure a virtual machine host with a single network interface
```

Comment:
This new document should also be linked from the "How-to guides" menu, by adding it to the `toctree` in `doc/howto.md`.

Reply:
Let's try to keep titles short. VM is an acceptable acronym.
```suggestion
# How to configure a VM host with a single network interface
```